pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/BinaryOperator.java

```diff
@@ -39,30 +38,30 @@ public String toString() {
      */
     public static BinaryOperator valueOf(BinaryExpression.Operator op) {
         switch (op) {
-        case ADDITION:
```

Comment:
missing indent consider JCC. might be pmd thing but could be a blindspot. Thats why CoC is so powerful, leveraging **open source solutions**.